packages/aragon-wrapper/src/index.js

```diff
@@ -213,6 +224,10 @@ export default class Aragon {
       codeAddress: values[2],
       isForwarder: values[3]
     }))
+
+    this.appProxyValuesCache[proxyAddress] = appProxyValues
```

Comment:
Shall we push these in an array instead, so that we can have a max limit of cached values?